release_docs/RELEASE.txt

```diff
@@ -200,6 +200,14 @@ New Features
     Library:
     --------
 
+    - Changed the default page buffer size for the ROS3 driver
+
+      Calling H5FDset_fapl_ros3() now has the side effect of setting the page
+      buffer size in the FAPL to 64MB if it was not previously set. This will
```

Comment:
Based on the page buffer's value for `ROS3_DEF_PAGE_BUF_SIZE` the units should be MiB, so "64MB" -> "64MiB".